notebooks/official/workbench/spark/spark_bigquery.ipynb

```diff
@@ -0,0 +1,1221 @@
+{
```

Comment:
<p>Please switch to UUID as TIMESTAMP has conflicts.</p><p> </p><p>See https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/745</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I saw that PR but haven't applied since it wasn't merged yet. Will change it!